features/orabos/file.include/etc/kernel/cmdline.d/30-hugepages.cfg

```diff
@@ -0,0 +1 @@
+CMDLINE_LINUX="$CMDLINE_LINUX default_hugepagesz=1G"
```

Comment:
We do not want to change the `hugepagesz`, the default (2MB) is fine.

Reply:
Just to follow up on the discussion we had recently:

On cascade lake we have:

- L1 TLB: 
  - 4x32 2MB Entries covering 128MB of memory
  - 1x4 1G entries covering 4GB of memory

It pretty much depends on the locality of your workload whether 2M or 1G pages are the better option. My suggestion would be the same as @fwiesel's. Let's stick with 2MB pages for the time being. 

When we have everything working, we can still do benchmarks to figure out whether 1G pages are actually helping, or not.